src/test/mocks/handlers.js

```diff
@@ -211,6 +211,14 @@ export const handlers = [
     const id = params['id'];
     return HttpResponse.json({ id: id });
   }),
+  http.post(
+    `${IMAGE_BUILDER_API}/experimental/blueprints/:id/fixup`,
+    ({ params }) => {
+      const id = params['id'];
+      getMockBlueprintResponse(id).lint.errors = [];
```

Comment:
This is a bit of a hack. I'm not sure if this is allowed, to write to the mocked data. I'm not sure if there's an alternative either.